src/main/java/org/apache/flink/benchmark/StateBackendBenchmark.java

```diff
@@ -91,7 +93,7 @@ public static void main(String[] args)
 
 	@Benchmark
 	public void stateBackends(StateBackendContext context) throws Exception {
-		IntLongApplications.reduceWithWindow(context.source, EventTimeSessionWindows.withGap(Time.seconds(500)));
+		IntLongApplications.reduceWithWindow(context.source, TumblingEventTimeWindows.of(Time.seconds(10_000)));
```

Comment:
These are probably only (mostly?) faster, because tumbling windows don't stress the state back-end that much with their own state (they only ever add values and then aggregate at the end) - session windows may constantly merge sessions and thus state as well. So I'm not sure, that by this change, you are actually getting the opposite, i.e. a higher part of state-unrelated overhead...(at least indirectly)

Reply:
The idea is that session windows are already so slow, that they can hide most of the cost of using rocks db. That's why @StephanEwen proposed to change it to "cheapest" windows.